src/OAuth2Client.php

```
@@ -35,47 +38,59 @@
     }
 
     /**
-     * @param  array{0: string, 1?: array<string, mixed>}  $parameters
+     * Magic method to handle HTTP requests.
+     *
+     * @param  string  $method  HTTP method name (get, post, put, patch, delete)
+     * @param  array  $parameters  [url, options]
      *
      * @throws OAuth2Exception
      */
     public function __call(string $method, array $parameters): Response
```

PHPStan (2.1.15)

Method Antogkou\LaravelOAuth2Client\OAuth2Client::__call() has parameter $parameters with no value type specified in iterable type array.
See: https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type

(missingType.iterableValue)